MapboxNavigation/RouteMapViewController.swift

```diff
@@ -40,6 +41,9 @@ class RouteMapViewController: UIViewController, PulleyPrimaryContentControllerDe
     var shieldImageDownloadToken: SDWebImageDownloadToken?
     var arrowCurrentStep: RouteStep?
     
+    let streetsLanguages = ["zh", "ru", "fr", "es", "en"]
+    let RoadLabelLayerIdentifier = "roadLabelLayer"
```

Comment:
This property is only used in one method, and there won’t ever be a need for another class to know about it. Make this a local variable instead.